frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/common/PortalConstants.java

```diff
@@ -37,7 +37,8 @@
     public static final Set<String> SET_CLEARING_TEAMS_STRING;
     public static final String LICENSE_IDENTIFIERS;
     public static final String PREFERRED_COUNTRY_CODES;
-    public static final Boolean MAINLINE_STATE_ENABLED_FOR_USER;
+    public static final boolean MAINLINE_STATE_ENABLED_FOR_USER;
+    public static final boolean IS_CLEARING_TEAM_UNKNOWN_ENABLED;
```

Comment:
Please revert the change for `MAINLINE_STATE_ENABLED_FOR_USER` as the change is not related. Also change the type of variable to `Boolean`  wrapper class.

Reply:
Done.